src/js/letters/containers/Main.jsx

```diff
@@ -26,10 +26,20 @@ export class Main extends React.Component {
     this.props.getAddressStates();
   }
 
+
+  appAvailability(lettersAvailability, addressAvailability) {
```

Comment:
This could probably be defined as a static method instead, but whatever

Reply:
Not sure I totally understand the suggestion, could you explain a bit more?

Reply:
Yeah sorry, that wasn't super helpful. I was terse because I really don't think this makes much difference, but here's a little more context if you're interested:
There's no need for this helper to have any understanding of what's going on inside a specific instance of `<Main />` because we're passing everything it needs via arguments (i.e., there's no need to call `this.whatever` from within the method). Defining as a static will keep the method on the prototype and you'd just use it by calling that way:
```
export class Main extends React.Component {
    static appAvailability(lettersAvail, addressAvail) { . . . }
    render() {
        switch(Main.appAvailability(
            this.props.lettersAvailability,
            this.props.addressAvailability
        ) { . . . }
    . . .
    }
};
```
Another option could be to define the method as a standalone function in the module but outside the class definition so it's clear that it's just a utility helper.

I mean, you could just leave it alone too. It's not like it doesn't work well the way it is 👍